ext/MitsubaLoader/CMitsubaLoader.cpp

```diff
@@ -532,7 +860,7 @@ CMitsubaLoader::SContext::shape_ass_type CMitsubaLoader::loadBasicShape(SContext
 	for (auto i=0u; i<mesh->getMeshBufferCount(); i++)
 		ctx.manipulator->flipSurfaces(mesh->getMeshBuffer(i));
 	// flip normals if necessary
-#define CRISS_FIX_THIS
+//#define CRISS_FIX_THIS
 #ifdef CRISS_FIX_THIS
```

Comment:
is it fixed?

Reply:
it's from before vulkan-like engine API. It was fixed, but it's not on this API/branch because we dont have mesh manipulator working

Reply:
we do have the mesh manipulator (normal recomputation) working IIRC ...

Reply:
maybe, not sure, but we dont have converter into unique primitives working

Reply:
can you bring it back quickly?

Reply:
probably, not sure, will look into it after fixing smaller issues

Reply:
needs to be back